ideascube/conf/idb_fra_aurore.py

```diff
@@ -1,28 +0,0 @@
-"""Ideasbox Aurore in Paris Clichy, France"""
-from .idb_fra import *  # pragma: no flakes
-
-
-IDEASCUBE_NAME = u"Aurore"
```

Comment:
> This can be removed as the user can now rename the server himself from UI

I know, I implemented that feature. :)

My point is that **right now**, they see "Aurore" in the top-right corner, and if we merely remove the configuration file they will suddenly see "Ideas Cube".

I think that's needlessly hostile towards our users, and I'm suggesting we run the command (maybe through Ansiblecube?) to set the name to what it currently is so that users don't see a difference after the upgrade.

The fact that it's configurable in the UI doesn't mean they will go and configure it, and we should try and limit disruptions whenever possible. This is a trivial one to avoid.

Did you already add the command in Ansiblecube and we can merge this without regrets? Or am I over-thinking this and we should not care?